app/src/main/java/com/pluu/webtoon/data/impl/AbstractWeekApi.kt

```diff
@@ -0,0 +1,27 @@
+package com.pluu.webtoon.data.impl
+
+import com.pluu.webtoon.data.WeeklyRequest
+import com.pluu.webtoon.item.Result
+import com.pluu.webtoon.item.ToonInfo
+import java.util.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/cac0e4180a6fb4c569235bdca8568373c67ba550/app/src/main/java/com/pluu/webtoon/data/impl/AbstractWeekApi.kt#L6"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>